server/router.js

```diff
@@ -95,4 +90,11 @@ function getDefaultAppState () {
   return state
 }
 
+function sendSPA (route, res, state) {
+  const frozenState = Object.freeze(state)
+  const contents = app.toString('/', frozenState)
```

Comment:
I can confirm the rehydration of state works!  but server-rendered /:archiveKey route is without the archive component.